doc/manual.rst

```diff
@@ -4247,16 +4247,38 @@ It is possible to raise/catch imported C++ exceptions. Types imported using
 caught by reference. Example:
 
 .. code-block:: nim
+    :test: "nim r -b:cpp $1"
```

Comment:
Please use more common: `nim cpp -r $1`

Reply:
done (but `nim r` has advantage of not clobbering your sources)